app/components/external-link.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: "a",
+  href: "#",
+
+  click: function() {
+    window.open(this.decodeLink(), "_system");
+    return false;
+  },
+
+  decodeLink: function(){
+    return this.attrs.linkUrl.replace(/&amp;/g, '&');
```

Comment:
@swatijadhav can we use a real url decoder here? Won't there be other characters apart from '&' that need escaping?